alpha/scripts/utils/updateEntriesRecordings.php

```diff
@@ -0,0 +1,105 @@
+<?php
+require_once('/opt/kaltura/app/tests/lib/KalturaClient.php');
+if ($argc < 6)
+{
+	die("Usage: php $argv[0] partnerId adminSecret serviceUrl simuliveEntryIdsFile vodEntryIdsFile \n");
+}
+$partnerId = $argv[1];
+$adminSecret = $argv[2];
+$url = $argv[3];
+$simuliveEntryIdsFile = $argv[4] ;
+$vodEntryIdsFile = $argv[5];
+
+$config = new KalturaConfiguration();
+$config->serviceUrl = $url;
+$client = new KalturaClient($config);
+$ks = $client->session->start(
+	$adminSecret,
+	'',
+	KalturaSessionType::ADMIN,
+	$partnerId,
+	86400,
+	'disableentitlement');
+$client->setKS($ks);
+
+$simuliveEntryIds = file ($simuliveEntryIdsFile) or die ('Could not read file'."\n");
+$vodEntryIds = file ($vodEntryIdsFile) or die ('Could not read file'."\n");
+
+if (count($simuliveEntryIds) != count($vodEntryIds))
+{
+	print_r("simuliveEntryIdsFile and vodEntryIdsFile should be on the same length \n");
+	exit;
+}
+
+for ($i = 0; $i < count($simuliveEntryIds); $i++)
+{
+	$simuliveEntryId = trim($simuliveEntryIds[$i]);
+	$vodEntryId = trim($vodEntryIds[$i]);
+	try
+	{
+		$simuliveEntry = $client->baseEntry->get($simuliveEntryId);
+	}
+	catch(Exception $e)
+	{
+		print_r($e->getMessage() . ' entryId: ' . $simuliveEntry ."\n");
+		continue;
+	}
+	if ($simuliveEntry)
+	{
+		$kLive = new KalturaLiveStreamEntry();
+		$kLive->recordedEntryId = $vodEntryId;
+		$kLive->displayInSearch = KalturaEntryDisplayInSearchType::SYSTEM;

```

Comment:
@ravitshalem  -  need to change the VOD entry displayInSearch to SYSTEM. Not the live entry. https://github.com/kaltura/mediaspace/blob/55af96d36294dad9d26394a3988a60d303ab7912/modulesCustom/core/eventplatform/reinventedit/models/Reinventedit.php#L150